app/src/app/(public)/properties/[id]/page.tsx

```
@@ -1,20 +1,38 @@
+    check_in?: string;
+    check_out?: string;
+  }>;
 }) => {
   const { id } = await params;
 
-  const property = await getProductQuery(Number(id));
+  const propertyId = Number(id);
+
+  const property = await getProductQuery(propertyId);
+
+  if (isNaN(propertyId)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)